src/lib/swapper/swappers/LifiSwapper/endpoints.ts

```diff
@@ -77,14 +82,14 @@ export const lifiApi: SwapperApi = {
     tradeQuote,
     supportsEIP1559,
   }: GetUnsignedEvmTransactionArgs): Promise<EvmTransactionRequest> => {
+    configureLiFi()
```

Comment:
Probably no need to configure it everywhere since this is a singleton and we could do it in a very early call in the app, but to avoid regressions and in the spirit of a sane diff, figured out that moving previous `getLifi()` (now `configureLiFi()` calls at the beginning of their respective bodies makes the most sense!